package/gluon-core/Makefile

```diff
@@ -2,7 +2,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gluon-core
 
-GLUON_VERSION = $(shell git describe --always --dirty=+ 2>/dev/null || echo unknown)
+GLUON_VERSION = $(shell cat $(TOPDIR)/../.scmversion 2>/dev/null || git describe --always --dirty=+ 2>/dev/null || echo unknown)
```

Comment:
Ugh, the current GLUON_VERSION code is quite ugly - we try to depend on the directory structure of Gluon + OpenWrt as little as possible. The `git describe` already breaks this rule, but accessing a file through `$(TOPDIR)/..` is even worse...

I'll send a PR that fixes this.